dashboard/modules/reporter/reporter_head.py

```diff
@@ -52,6 +53,42 @@ async def launch_profiling(self, req) -> aiohttp.web.Response:
             message="Profiling success.",
             profiling_info=profiling_info)
 
+    @routes.get("/api/ray_config")
+    async def get_ray_config(self, req) -> aiohttp.web.Response:
```

Comment:
If the `~/ray_bootstrap_config.yaml` is not changed after ray is bootstrapped, we can just read it once. There are many ways:
- Add a `@aiohttp_cache` decorator with a long TTL: https://github.com/ray-project/ray/pull/10921
- Save the read `payload` to `self._config`, if the `self._config` is not None just return it.